templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -97,6 +99,17 @@
                 [% END %]
 
             [% END %]
+        </form>
```

Comment:
That's strange, if you put `</form>` here, don't you get inputs, buttons etc. that are outside the `<form>..</form>` ?